src/PointCloud/Potree/V2/Potree2Reader.cs

```diff
@@ -258,9 +258,44 @@ public PotreeData ReadNewFile(string path)
 
             CacheMetadata(true);
 
+            PotreeData.Metadata.PrincipalAxisRotation = CalculatePrincipalAxis(PotreeData);
```

Comment:
Should we make this optinal in `ReadNewFile` (`default = false`)?

Reply:
Checked. This is unfortunately not a promising idea, as we have the method `ReadNewFile`, as well as the `ctor` which uses these methods (and would need a `default = false`, too). Furthermore, we are bound to run into problems during export and import and settings which can be changed on the fly like orient point cloud or do not orient. However, if we loaded with no axis, we could never activate this option without a crash. During the write operation we un- and reload the pointcloud. There we would need a flag, too, if we need to calculate the axis or not, etc.